wiki/Katu/en.md

```diff
@@ -9,6 +8,56 @@ tags:
 
 *See also: [Geki](/wiki/Geki)*
 
-**Katu (喝)**, *Katsu*, or *Beat!*, is a [scoring](/wiki/Score) term used when a player has completed a comboset without achieving the highest possible [accuracy](/wiki/Gameplay/Accuracy) on every note. This idea originates from the Nintendo DS game Elite Beat Agents which is one of the games the [osu!standard](/wiki/Game_mode/osu!) [game mode](/wiki/Game_mode) is based on.
+*Not to be confused with osu!taiko's Katu*
 
-<!-- TODO: Add links -->
+**Katu (喝)**, *Katsu*, or *Beat!*, is a [scoring](/wiki/Score) term used when a player has completed a comboset without achieving the highest possible [accuracy](/wiki/Gameplay/Accuracy) on every note.
```

Comment:
the same thing that I pointed out in "Geki": please use "combo" instead of "comboset"